docs/variants/dell_optiplex/firmware-update.md

```diff
@@ -10,10 +10,14 @@ distribution update.
 For simplicity we recommend using network booted
 [Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite).
 
-### Dasharo (coreboot+SeaBIOS) update
+### Dasharo (coreboot+SeaBIOS) and Dasharo (coreboot+UEFI) update
 
 * Make sure a wired network cable to the device's Ethernet port
-* Boot platform and from SeaBIOS menu choose Dasharo Network Boot Menu
+* Perform DTS network boot
+  * Dasharo (coreboot+SeaBIOS)
+    * SeaBIOS menu choose Dasharo Network Boot Menu
```

Comment:
How does it work? Shouldn't there be an iPXE? And then in the iPXE menu I would expect such entry

Reply:
Fixed https://github.com/Dasharo/docs/pull/336/commits/c006097c2b665a08652d2089e8cc0edd2a3cbb6b